crates/red_knot_python_semantic/src/types.rs

```diff
@@ -355,6 +355,8 @@ pub enum Type<'db> {
     Instance(InstanceType<'db>),
     /// A single Python object that requires special treatment in the type system
     KnownInstance(KnownInstanceType<'db>),
+    /// A type alias, with a name and corresponding type
```

Comment:
Should clarify what kind, since there are also legacy type aliases.
```suggestion
    /// A PEP 695 type alias, with a name and corresponding type
```